flatnav_python/setup.py

```diff
@@ -49,12 +49,11 @@ def simd_extension_supported(extension: str) -> bool:
     "-funroll-loops",  # Unroll loops
 ]
 
-# We don't include SIMD flags if the user has set the NO_MANUAL_VECTORIZATION
-# environment variable to 1
-no_manual_vectorization = int(os.environ.get("NO_MANUAL_VECTORIZATION", "0"))
+# We don't include SIMD flags if the NO_SIMD_VECTORIZATION variable is set to 1
+no_simd_vectorization = int(os.environ.get("NO_SIMD_VECTORIZATION", "0"))
 
-if not no_manual_vectorization:
-    SIMD_EXTENSIONS = ["sse", "avx", "avx512f"]
+if not no_simd_vectorization:
+    SIMD_EXTENSIONS = ["sse", "sse3", "avx", "avx512f"]
```

Comment:
Added `sse3` here because I added an implementation that requires the `_mm_hadd_ps` intrinsic for summing all floats in a 128-bit register, which is only available in SSE3 and not SSE/SSE2.